docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -70,12 +70,15 @@ Figure 4-30: Docker Tools Options
 
 The following table might help you decide how to set these options.
 
-| Name | Default Setting | Description |
-| -----|:---------------:| ----------- |
-| Automatically pull required Docker images on project load | On | For increased performance, when loading projects, Visual Studio will start a Docker pull operation in the background so that when you are ready to run your code, the image is already downloaded or in the process of downloading. If you're just loading projects and browsing code, you can turn this off to avoid downloading container images you don't need. |
-| Automatically start containers in background | On | Again for increased performance, Visual Studio creates a container with volume mounts ready for when you build and run your container. If you want to control when your container is created, turn this off. |
-| Automatically kill containers on solution close | On | Turn this off if you would like containers for your solution to continue to run after closing the solution or closing Visual Studio. |
-| Do not prompt for trusting localhost SSL certificate | Off | Visual Studio will continue to prompt for new projects even after the SSL certificate has been trusted for an earlier project. You can set this checkbox to avoid getting the prompt when you open other projects. |
+| Name | Default Setting | Applies To | Description |
+| -----|:---------------:|:----------:| ----------- |
+| Automatically pull required Docker images on project load | On | Docker Compose | For increased performance, when loading projects, Visual Studio will start a Docker pull operation in the background so that when you are ready to run your code, the image is already downloaded or in the process of downloading. If you're just loading projects and browsing code, you can turn this off to avoid downloading container images you don't need. |
+| Automatically start containers in background | On | Docker Compose | Again for increased performance, Visual Studio creates a container with volume mounts ready for when you build and run your container. If you want to control when your container is created, turn this off. |
+| Automatically kill containers on solution close | On | Docker Compose | Turn this off if you would like containers for your solution to continue to run after closing the solution or closing Visual Studio. |
+| Do not prompt for trusting localhost SSL certificate | Off | ASP.NET Core 2.1 projects | If the localhost SSL certificate is not trusted, Visual Studio will prompt every time you run your project, unless this checkbox is checked. |
+
+> [!WARNING]
+> If the localhost SSL certificate is not trusted, and you check the box to suppress prompting, then your project can't run. In that case, uncheck the **Do not prompt** checkbox, run your project, indicate trust at the prompt.
```

Comment:
an "and" should separate the comma and "indicate"

Reply:
Thanks